wiki/Installing-Oppia-Android.md

```diff
@@ -81,36 +84,40 @@ Please follow these steps to set up Oppia Android on your local machine.
 ## Set up and run tests
 Testing the app is an integral part of our development process. You will need to test all code changes to ensure that the app works correctly, therefore it is important to ensure that your test configuration works.
 
-We run tests in either Espresso(`app` module tests) or Robolectric(non-app module tests), meaning tests will run in either the emulator or on Gradle/Bazel via the terminal.
+Robolectric allows for faster unit testing of Android applications without the need for device or emulator setup.
```

Comment:
```suggestion
We strongly recommend running tests on Robolectric which is faster because it does not require a physical device or emulator setup.
```

Reply:
Done.